src/content/docs/merchant/multistore.mdx

```diff
@@ -0,0 +1,185 @@
+
+Content is organized primarily by language, enabling global reuse and minimizing duplication. US English files should be placed in the `en` directory of the project root. All other language- and region-specific files should be placed in a directory that is named in the format `*region-code_language-code*`. For example, the `/en/` directory contains data for the US market, while the  `/en_ca/` directory contains data for Canada. French content should be placed in directories with names like `fr_fr` for France and `fr_ca` for Canada.
+
+The Sharepoint file structure of a multi-store is shown below. Remove the `.docx` and `.xlsx` extensions for Google Drive files.
+
+<FileTree>
+- **en/** _-- English store (Default root language)_
+  - configs.xls _-- Defines API endpoints, headers, and other environment-specific data._
+  - placeholders.xlsx _-- Stores reusable tokens for text and UI components._
+  - index.docx _-- The home page of the English US store._
+- **en_ca/** _-- English Canadian Store_
+  - configs.xls _-- Environment specific data for the English Canadian Store_
+  - placeholders_overrides.xlsx _-- Entries that override the English store._
+  - index.docx _-- The home page of the English Canadian store._
+- fstab.yaml _-- Sets mountpoints and folder mappings._
+- metadata.xlsx _-- URL, root path, and placeholder mappings for multi-store_
+- store-switcher.docx _-- Manages the list of stores and their URL to render in the UI_
+</FileTree>
+
+Default files are provided in the example content. These files can be customized to suit your needs.
+
+## Global files
+
+These files reside in the root of the project and are shared across all stores.
+
+### Metadata
+
+Metadata entries are managed as bulk (https://www.aem.live/docs/bulk-metadata) in `metadata.xlsx`.
+This file overrides Commerce endpoint config files and placeholders based on the `root` page metadata provided. Otherwise, it defaults to the base `/`.
+
+From the root of your project, create a `metadata.xlsx` file with contents similar to the following:
+
+| URL | Root | Placeholders |
+|---|---|---|
+| /en/** | /en/ | /en/placeholders.json |
+| /en_ca/** | /en_ca/ | /en/placeholders.json<br/>/en_ca/placeholders-overrides.json |
+
+The URL value defines the path to the store view. The root value specifies the directory where the `configs.xlsx` files are located. The placeholders value specifies the `placeholders.json` file to use. Adding a new line with another path in this value specifies an override file. Both files will be fetched, and the values in the second file will override those in the first.
+
+### Store Switcher
+
+The `store-switcher.docx` file manages the list of stores that are displayed in a store switcher element.
+This file contains a bulleted list that can be nested. Each line must define the display name of your store with an active link to the store.
+
```

Comment:
@keharper @anthoula I suggest putting in the format of what the store-switcher.docx should look like here. 

Select a store  

 

--- 
United States ($) 
[United States ($)](https://main--aem-boilerplate-commerce--hlxsites.aem.page/drafts/multistore/en/)  

Canada (CA$) 
[Canada (EN)](https://main--aem-boilerplate-commerce--hlxsites.aem.page/drafts/multistore/en_ca/) 

[Canada (FR)](https://main--aem-boilerplate-commerce--hlxsites.aem.page/drafts/multistore/en_ca/) 

---